packages/generate-pdf/test/generate-pdf.spec.ts

```diff
@@ -248,6 +252,42 @@ const runPDFGenerationTests = (testSuite: PDFGenerationTestProperties) => {
       expect(resultHash).toEqual(expectedHash);
     }, 10000);
 
+    it('should attach the certificate as JSON document if `attachCertificate` is true', async () => {
+      const translations = JSON.parse(readFileSync(translationsPath, 'utf8'));
+      (attachFileToPdf as jest.Mock).mockClear();
```

Comment:
ideally you could run `(attachFileToPdf as jest.Mock).mockClear();` in `afterEach` hook